pkg/config/manifest.go

```diff
@@ -20,7 +20,8 @@ type Manifest struct {
 	path    string
 	outputs map[string]string
 
-	Name         string                 `yaml:"image,omitempty"`
```

Comment:
Looks like image is no longer being populated anymore?

Reply:
Indeed, if there was an `image` field in the example [porter.yaml](https://github.com/deislabs/porter/blob/master/examples/azure-mysql-wordpress/porter.yaml) at one point, it is no longer there.  I've assumed this should point to the `name` field, which is populated.